sdk/identity/Azure.Identity/BREAKING_CHANGES.md

```diff
@@ -26,6 +26,8 @@ var credential = new DefaultAzureCredential(new DefaultAzureCredentialOptions
 
 Note: Credential types which do not require a `TenantId` on construction will only throw `AuthenticationFailedException` when the application has provided a value for `TenantId` either in the options or via a constructor overload. If no `TenantId` is specified when constructing the credential, the credential will acquire tokens for any requested `TenantId` regardless of the value of `AdditionallyAllowedTenants`.
 
+More information on this change and the consideration behind it can be found [here](https://aka.ms/azsdk/blog/multi-tenant-guidance).
```

Comment:
This link doesn't appear to work yet.

Reply:
By design. Scott mentioned that the aka.ms will be live once the blogs go live.